pandas/core/arrays/base.py

```diff
@@ -532,6 +517,7 @@ def dtype(self) -> ExtensionDtype:
 
     @property
     def shape(self) -> Shape:
+
```

Comment:
also, please check that you don't accidentally introduce extra blank lines

Reply:
Sorry about that will change.